packages/evm/contracts/payments/PaymentsBook.sol

```diff
@@ -0,0 +1,535 @@
+    * @param _periodId Identification number of the period being queried
+    * @param _tokens List of token addresses to be queried
+    * @return List of token amounts corresponding to the governor
+    */
+    function getManyGovernorFees(uint256 _periodId, address[] calldata _tokens) external view returns (uint256[] memory amounts) {
+        Period storage period = periods[_periodId];
+
+        amounts = new uint256[](_tokens.length);
+        for (uint256 i = 0; i < _tokens.length; i++) {
+            amounts[i] = _getGovernorFees(period, _tokens[i]);
+        }
+    }
+
+    /**
+    * @dev Internal function to claim juror fees for a certain period
+    * @param _period Period being claimed
+    * @param _periodId Identification number of the period claiming fees for
+    * @param _juror Address of the juror claiming the fees
+    * @param _token Address of the token being claimed
+    * @param _amount Amount of tokens to be transferred to the juror
+    */
+    function _claimJurorFees(Period storage _period, uint256 _periodId, address payable _juror, address _token, uint256 _amount) internal {
+        if (_amount > 0) {
+            _period.claimedJurorFees[_juror][_token] = true;
+            _transfer(_juror, _token, _amount);
+            emit JurorFeesClaimed(_periodId, _juror, _token, _amount);
+        }
+    }
+
+    /**
+    * @dev Internal function to transfer governor fees for a certain period
+    * @param _period Period being claimed
+    * @param _periodId Identification number of the period being claimed
+    * @param _token Address of the token to be claimed
+    */
+    function _transferGovernorFees(Period storage _period, uint256 _periodId, address payable _governor, address _token) internal {
+        uint256 amount = _getGovernorFees(_period, _token);
+        if (amount > 0) {
+            _period.governorFees[_token] = 0;
+            _transfer(_governor, _token, amount);
+            emit GovernorFeesTransferred(_periodId, _token, amount);
+        }
+    }
+
+    /**
+    * @dev Internal function to pull tokens into this contract
+    * @param _from Owner of the deposited funds
+    * @param _token Address of the token to deposit
+    * @param _amount Amount to be deposited
+    */
+    function _deposit(address _from, address _token, uint256 _amount) internal {
+        if (_token == address(0)) {
+            require(msg.value == _amount, ERROR_ETH_DEPOSIT_MISMATCH);
+        } else {
+            require(ERC20(_token).safeTransferFrom(_from, address(this), _amount), ERROR_TOKEN_DEPOSIT_FAILED);
+        }
+    }
+
+    /**
+    * @dev Internal function to transfer tokens
+    * @param _to Recipient of the transfer
+    * @param _token Address of the token to transfer
+    * @param _amount Amount to be transferred
+    */
+    function _transfer(address payable _to, address _token, uint256 _amount) internal {
+        if (_token == address(0)) {
+            // solium-disable-next-line security/no-send
```

Comment:
We might as well just disable this rule in the config, since it's now an anti-practice to use `.transfer()`.

Reply:
Addressed in ee7d2b6144d821812d9ef6ac2f7451288238a17a